src/Microsoft.AspNetCore.Razor.Language/Intermediate/IntermediateNodeBuilder.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.AspNetCore.Razor.Language.Intermediate
 {
-    public abstract class IntermediateNodeBuilder
+    internal abstract class IntermediateNodeBuilder
```

Comment:
@NTaylorMullen, I'm thinking I'll keep this how it is right now unless you feel strongly otherwise.